ivy/functional/backends/jax/statistical.py

```diff
@@ -37,7 +37,7 @@ def max(
     return jnp.max(a=jnp.asarray(x), axis=axis, keepdims=keepdims)
 
 
-@with_unsupported_dtypes({"0.4.14 and below": "bfloat16"}, backend_version)
+@with_unsupported_dtypes({"0.4.19 and below": "bfloat16"}, backend_version)
```

Comment:
I think this should be `("bfloat16",)` instead of `"bfloat16"` for it to be considered a `tuple`